clients/unshaded/src/main/java/tachyon/client/TachyonFSTestUtils.java

```diff
@@ -124,9 +124,9 @@ public static TachyonFile createByteFile(TachyonFileSystem tfs, String fileName,
     List<FileInfo> infos = tfs.listStatus(tfs.open(new TachyonURI(path)));
     List<String> res = new ArrayList<String>();
     for (FileInfo info : infos) {
-      res.add(info.getPath());
-
-      if (info.isFolder) {
+      if (!info.isFolder) {
+        res.add(info.getPath());
+      } else {
         res.addAll(listFiles(tfs, info.getPath()));
       }
```

Comment:
Ok. So if it should just return files and not directories, that should be updated in the method javadocs. And, it looks like it lists all files recursively, so that should also be documented.

On a side note, how did this work before?